.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,56 @@
+name: Pull Request for orders
+on:
+  push:
+    branches:
+      - orders
```

Comment:
why only for orders branches?! If I create a branch named test1 your action will not be triggered.
Make it generic. BTW there is one exception!!! One branch should be excluded! Which one?! :)

Reply:
you do not need GITHUB_REF_NAME if you specify pattern for allowed/not allowed branches

Reply:
know, i use it for title of PR only. i fixed , action will work for any branch without master